### ISSUE TITLE
Datamodel Sort broken in 1.3.1

### ISSUE THREAD

**Issue by rodmaz** [chartshq/muze#280]
**Do you want to request a *feature* or report a *bug*?**
Reporting a bug.

**What is the current behavior?**
Using data model `sort` function used to work in version 1.2.1 but is broken in version `1.3.1`.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code. You can either a scale down sample, JSFiddle or JSBin link**

**What is the expected behavior?**
A bar chart with sorting enabled (e.g. `percentage`) should sort the columns according to a dimension:

```
            let dm = new DataModel(data, schema);
            dm = dm.sort([
              ['percentage', 'ASC'],
            ]);
```
Now regardless the way we specify, it disregard the sort.

<img width="607" alt="Screen Shot 2020-02-12 at 16 26 49" src="https://user-images.githubusercontent.com/55300/74369661-82ac3f80-4db4-11ea-85b4-c94a15a2208f.png">

**Which versions of MuzeJS, and which browser/OS are affected by this issue? Did this work in previous versions of MuzeJS?**
Muze 1.3.1, macOS 10.15.3, Chrome 79.0.3945.130

**Comment by UD-UD** [comment]
Thanks for the bug report.

We will be taking this in our next iteration.And it will be available as part of `v1.3.3` release.

Till then you can have the same expected behaviour by using one of Muze's API as follows :
```
.config({
      sort: {
        <FIELD_NAME>: "desc | asc"
      }
    })
```
Also we have created an [CODEPEN](https://codepen.io/adarshlilha/pen/poJjmdB) example for the same.

Hope this helps.

**Comment by rodmaz** [comment]
Hi, why has this issue been closed? Is version 1.3.3 out?

**Comment by UD-UD** [comment]
@rodmaz This fix is now available as part of our new performant library `@chartshq/muze`  
Lets us know if you face any issue.
Here is an [example](staging.muzejs.org/demos/wa/latest/horizontal-bar-chart) for the same.
